src/command/root-command/index.ts

```diff
@@ -30,57 +23,32 @@ export class RootCommand {
   @ExternalOption('quiet')
   public quiet!: boolean
 
-  // CLASS FIELDS
-
-  public appName = 'swarm-cli'
-
   public bee!: Bee
-
   public beeDebug!: BeeDebug
-
-  public commandConfig!: CommandConfig
-
   public console!: CommandLog
+  public appName = 'swarm-cli'
+  public commandConfig!: CommandConfig
 
-  /**
-   * Init Root command fields
-   *
-   * if BEE_API_URL environment variable has been set the CLI will use that connection string
-   */
   protected init(): void {
-    // Console logs of default commands
-    this.verbosity = VerbosityLevel.Normal
-
-    if (this.optionPassed(quiet)) {
-      this.verbosity = VerbosityLevel.Quiet
-    }
-
-    if (this.optionPassed(verbose)) {
-      this.verbosity = VerbosityLevel.Verbose
+    this.commandConfig = new CommandConfig(this.appName, this.console, this.configFolder)
+    const sourcemap = Utils.getSourcemap()
+    if (sourcemap[beeApiUrl.key] === 'default' && this.commandConfig.config.beeApiUrl) {
```

Comment:
We have a linter rule to enforce blank lines before `if` statements. 

Now this is subjective but I could also imagine having more newlines inserted in general to separate and group lines of code according to their functionality.

Reply:
I don't always turn ESLint off temporarily, but when I do, I surely forget to turn it back on before pushing code 🙂